CONTRIBUTE.md

```diff
@@ -4,5 +4,19 @@
 
 - fork
 - clone fork
+- start a new branch `git branch my-contrib`
+- switch to the new branch `git checkout my-contrib`
```

Comment:
- start a new branch `git branch my-contrib`
- switch to the new branch `git checkout my-contrib`
could be replaced with:
Create a new branch and switch to the new branch `git checkout -b my-contrib`